12_huggingface_keras_classifier/requirements.txt

```diff
@@ -0,0 +1,6 @@
+ipywidgets==8.0.2
+numpy>=1.22.0
+scikit_learn==1.0.2
+tensorflow==2.9.2 # for Macs with Apple silicon : tensorflow-macos==2.9.2 and tensorflow-metal==0.5.1
+tensorflow-datasets==4.6.0
+transformers
```

Comment:
can we also add version for this?

Reply:
I think it's ok if `transformers` version here doesn't exactly match the odd `4.22.0.dev0` one specified in the other notebook.

Reply:
Added! (precisely left it out on purpose earlier because it did not match the `4.22.0.dev0` one)